alerts/plugins/port_scan_enrichment.py

```diff
@@ -196,10 +196,10 @@ def enrich(alert, search_fn, search_window, max_connections):
         for hit in results.get('hits', [])
     ]
 
-    alert['details']['recentconnections'] = []
```

Comment:
I think we still want to save the recentconnections array on alerts['details'], not the root.